stubs/networkx/networkx/utils/configs.pyi

```diff
@@ -1,26 +1,64 @@
+import sys
 from _typeshed import Incomplete
-from collections.abc import ItemsView, Iterable, Iterator, KeysView, Mapping, ValuesView
+from collections.abc import Callable, ItemsView, Iterable, Iterator, KeysView, ValuesView
+from dataclasses import dataclass
+from types import TracebackType
 from typing_extensions import Self
 
 __all__ = ["Config"]
 
-class Config(Mapping[str, Incomplete]):
-    def __init_subclass__(cls, strict: bool = True) -> None: ...
-    def __new__(cls, **kwargs) -> Self: ...
-    def __dir__(self) -> Iterable[str]: ...
-    def __setattr__(self, name: str, value) -> None: ...
-    def __delattr__(self, name: str) -> None: ...
-    def __contains__(self, key: object) -> bool: ...
-    def __iter__(self) -> Iterator[str]: ...
-    def __len__(self) -> int: ...
-    def __reversed__(self) -> Iterator[str]: ...
-    def __getitem__(self, key: str): ...
-    def __setitem__(self, key: str, value) -> None: ...
-    def __delitem__(self, key: str) -> None: ...
-    def get(self, key: str, default: Incomplete | None = None): ...
-    def items(self) -> ItemsView[str, Incomplete]: ...
-    def keys(self) -> KeysView[str]: ...
-    def values(self) -> ValuesView[Incomplete]: ...
+# Our pyright test doesn't understand `requires_python` in METADATA.toml
```

Comment:
I'm not too happy about this hack. Let's at least add a "TODO" comment and link a typeshed issue to fix our pyright test.